### PR TITLE
feat(environments): create environment route with sub-routes

### PR DESCRIPTION
**Pull request by lorenzo-cavazzi**
This PR integrates the "launch notebook" page into the new "Environments" tab, applying also minor changes to the layout according to #482. Please note that it doesn't filter/change the environment options (refer to the issue).

Preview: https://lorenzotest.dev.renku.ch/

Screenshot from 2019-09-17 08-54-59

### CODE REVIEW

**Comment by lorenzo-cavazzi** [comment]
It turns out that the ordering of the options returned by the server is different from the one defined in https://github.com/SwissDataScienceCenter/renku-notebooks/blob/e185002467c75954d749e29e3a7c5f55574257d6/helm-chart/renku-notebooks/values.yaml#L50

After digging a little bit into the details, it seems the responsible is the helm `toJson` function here  https://github.com/SwissDataScienceCenter/renku-notebooks/blob/e185002467c75954d749e29e3a7c5f55574257d6/helm-chart/renku-notebooks/templates/configmap.yaml#L13

We could add an `ordering` properties to the `serverOptions` properties to help the UI rendering options in the desired order.

**Comment by lorenzo-cavazzi** [comment]
The first request is reasonable, but it's currently not easy to implement a clean solution. The main problem is that information about notebooks is available only inside a notebook container component: they instantiate a "private" react store and poll the resource independently, so adding a `<StartNotebookServer>` component inside `<Notebooks>` will end up in polling `/notebooks` api twice.
Changing it is not trivial, we could implement a not-so-elegant workaround but I believe we should create an issue (epic?) to address this problem in a coherent way for every redux store we are using in the UI.

We may go for another solution
* redirect to the `/environments/new` page after the notebooks' data are first loaded -- if this happens after the spinning wheel, it should be (almost) transparent to the user
* add a link to click to go to the `new` page -- this will still require an extra click but it will be less confusing
* ... any further idea?

**Comment by lorenzo-cavazzi** [comment]
I implemented a solution for all the 3 topics and updated the deployment in https://lorenzotest.dev.renku.ch

Should I create a new issue out of the first request about no running environments? (display the "new" page instead of "running"?)

File: src/project/Project.present.js [hunk 1]
```diff
@@ -601,21 +601,44 @@ function notebookLauncher(userId, accessLevel, notebookLauncher, fork, postLogin
...
+      <Col key="nav" xs={12} md={2}>
+        <Nav pills className="flex-column mb-3">
+          <NavItem>
+            <RenkuNavLink to={this.props.notebookServersUrl} title="Running" />
+          </NavItem>
+          <NavItem>
+            <RenkuNavLink to={this.props.launchNotebookUrl} title="New" />
+          </NavItem>
+        </Nav>
+        {/* <ProjectViewOverviewNav {...this.props} /> */}
```
Comment (ciyer) [inline src/project/Project.present.js]: Was this intended to be removed? If it is supposed to be here, maybe add a comment explaining why.

Comment (lorenzo-cavazzi) [reply src/project/Project.present.js]: I forgot it there, will remove it :slightly_smiling_face:

**Pull request merged**